src/admin/dumb_components/sub_due_form.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+const { reduxForm } = require('redux-form')
+
+const sub_due_dates = ({fields: {start, end}, handleSubmit, error}) =>
+  <form className='date-boundaries-form' onSubmit={handleSubmit}>
+    <input type='text' placeholder='Start date' {...start}/>
+    {start.touched && start.error && <div>{start.error}</div>}
+    <input type='text' placeholder='End date' {...end}/>
+    {start.touched && start.error && <div>{start.error}</div>}
```

Comment:
Can't believe I just spent 15 minutes looking for .touched...

Now it is working I will look into the refactor.